src/Microsoft.ML.AutoML/TrainerExtensions/MultiTrainerExtensions.cs

```diff
@@ -240,7 +240,6 @@ internal class ImageClassificationExtension : ITrainerExtension
             ColumnInformation columnInfo, IDataView validationSet)
         {
             var options = TrainerExtensionUtil.CreateOptions<Options>(null, columnInfo.LabelColumnName);
-            options.ValidationSet = validationSet;
```

Comment:
Are you also passing the validation dataset during CodeGen? If so, it should be removed there too.

Reply:
No, I'm not passing validations set at codegen.